_targets/build.project.armv7m7-imxrt106x

```diff
@@ -95,34 +94,36 @@ b_build_target() {
 
 	b_log "Relink plo to include the new script"
 	(cd plo && make $MAKEFLAGS all)
+
+	cp "${PREFIX_PROG_STRIPPED}plo-ram-${TARGET}.img" _boot/
 }
 
 
 b_add2img() {
-	printf "Copying %s (offs=%dB)\n" $1 $OFFSET
-	dd if=$1 of=$2 bs=1 seek=$OFFSET conv=notrunc 2>/dev/null
-	SZ=$((($(stat -c "%s" $1) + $SIZE_PAGE - 1)&$PAGE_MASK))
-	OFFSET=$(($OFFSET+$SZ))
+	printf "Copying %s (offs=%dB)\n" "$1" "$OFFSET"
+	dd if="$1" of="$2" bs=1 seek="$OFFSET" conv=notrunc 2>/dev/null
+	SZ=$((($(stat -c "%s" "$1") + SIZE_PAGE - 1) & PAGE_MASK))
+	OFFSET=$((OFFSET + SZ))
 }
 
 
 b_image_target() {
 	b_log "Creating image"
 
 	IMG=_boot/phoenix-${TARGET}.disk
```

Comment:
Is this a NOR FLASH image? If so, I think:
 - all unused bytes should be initialized to `0xFF`
 - the flash image should have fixed size (equal to NOR partition/whole flash size)